hw/dv/tools/rules.mk

```diff
@@ -53,12 +53,13 @@ ifneq (${SW_NAME},)
 	# NOTE: We're using the fpga platform for now, because there is no
 	# such thing as a DV platform yet (nor does any code do anything
 	# special for DV yet).
-	ninja -C $(SW_BUILD_DIR)/meson/build-out/sw/fpga all
+	ninja -C ${PROJ_ROOT}/build-out sw/device/boot_rom/boot_rom_export_${SW_BUILD_SUFFIX}
```

Comment:
I changed this away from `all` just in case in the future the default is no longer DV friendly because of some environmental differences.  I figure it's better to be specific.

Reply:
Yup that sounds good.

Reply:
Can we modify this command such that the cp commands below are no longer required?

Reply:
i could modify it so that the build root is directly the scratch area.  But the meson build output name is still unique, and i think we need something generic for the tb load.

I'm not sure if there's a way to pass an option into meson to ask it to use a specific name and output directory (as of right now)